docs: update README.md (#88)
instill-ai/vdp: README.md

Execute the following commands to start pre-built images with all the dependencies:

```bash
$ git clone https://github.com/instill-ai/vdp.git && cd vdp

# Build instill/vdp:dev local development image

> $ make build

Reviewer: Do we need to add `make build` for Quickstart? I thought `make all` will cover it?

Author: We probably don't want to build `instill-ai/vdp:dev` everytime when `make all` so `make build` is a step alone now.

# Launch all services.
$ make all
```

> :warning: The download of the Triton server image will take a while, but the image pulling process should be just a one-time effort.

Reviewer: ```suggestion
:warning: Downloading the Triton server image will take a while, but it should be just a one-time effort.
```
